draft-ietf-jsonpath-base.md

```diff
@@ -1471,7 +1471,7 @@ according to one of the following rules:
 ### `length` Function Extension {#length}
```

Comment:
The text of this section still assumes that the argument is a value.  The list of cases works out, but this is still a bit inconsistent.

Reply:
Hmmm. It seemed consistent with descriptions of other functions which take optional arguments.

Reply:
let's go for it for now, we can take another pass.

Reply:
> Its only argument is a value (possibly taken from a singular path as
> in the example above).  The result also is a value, an unsigned
> integer.

It uses "value" language for both input and output.  Is it safe to assume from 

> For any other argument value, the result is `Nothing`.

and typing it as `OptionalValue`, that `Nothing` is returned when `Nothing` is received?

Reply:
I think that's pretty clear, yes. I've tried a clarification, but we can tweak later if necessary.